cli/test_command_apikey_operations.bats

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bats
+
+setup() {
+	if [[ -e /tmp/apikeys.json ]]; then
+		rm /tmp/apikeys.json
+	fi
+}
+
+teardown() {
+	if [[ -e /tmp/apikeys.json ]]; then
+		rm /tmp/apikeys.json
+	fi
+}
+
+@test "load command works and is idempotent" {
+	run st2 apikey create
+	[[ "$status" -eq 0 ]]
```

Comment:
I assume that's some kind of bats "magic" when you use ``run`` it saves exit code in ``$status`` and stdout/stderr in ``$output``?

Reply:
It looks like it might also be a good idea to use bats-assert library - https://github.com/ztombol/bats-assert

Reply:
That's exactly what it does - both `$status` and `$output` are automatically set by BATS (`$output` contains stderr and stdout concatenated together, in that order).

And thanks for the pointer to bats-assert. I've started using it and I really like it so far. I'll convert all asserts to use it just to remain consistent.

One concern I have with it is installing bats-assert and bats-support (a dependency of bats-assert) before the test, but that's a solvable problem.

Reply:
Rewrote asserts to all use asserts from bats-assert.